tailcfg/tailcfg.go

```diff
@@ -185,6 +192,20 @@ type Node struct {
 	MachineAuthorized bool `json:",omitempty"` // TODO(crawshaw): replace with MachineStatus
 }
 
+// DefaultDisplayName generates the default title for n.
```

Comment:
Let's not say "title" since that's not defined and begs more questions.

// DefaultDisplayName returns a value suitable for using as the default value for n.DisplayName.